File: Models/Pedido.cs
namespace CardapioOnline.Models;

using System.ComponentModel.DataAnnotations;

public class Pedido
{
    public long Id { get; set; }
    public string NomeCliente { get; set; }
    public DateTime DataPedido { get; set; }
    public int NumeroMesaCliente { get; set; }
    public bool ChamadoGarcom { get; set; } = false;
    public List<DetalhePedido> DetalhesPedido { get; set; } = new List<DetalhePedido>();
}

File: front-cardapio-online/src/interfaces/Pedido.ts
import DetalhePedido from "./DetalhePedido";

export default interface Pedido {
  id: number;
  nomeCliente: string;
  dataPedido: string; // ISO string
  numeroMesaCliente: number;
  chamadoGarcom: boolean;
  detalhesPedido: DetalhePedido[];
}
